### PR TITLE
Test 'current environment settings object' for Worker() URL parsing and COEP report

### PR DESCRIPTION
**Pull request by zcorpan**
See https://github.com/whatwg/html/pull/6525

cc @d0iasm

### CODE REVIEW

**Comment by zcorpan** [comment]
I don't think it's possible to test the difference between "current environment settings object" and "relevant environment settings object" when it's a constructor, since you can't do `foo.call(otherWindow.foo)` to invoke the constructor. It's possible I'm missing something though.

File: workers/multi-globals/entry.html [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+
+<iframe src="incumbent/incumbent.html"></iframe>
+<button onclick="frames[0].hello()" id="button">Hello</button>
+
+<script>
+async_test((t) => {
+  onload = () => {
+    document.getElementById('button').click();
```
Comment (domenic) [inline workers/multi-globals/entry.html:13]: The indirection through the button click shouldn't be necessary. Compare to e.g.

- https://github.com/web-platform-tests/wpt/blob/master/fetch/api/request/multi-globals/url-parsing.html
- https://github.com/web-platform-tests/wpt/blob/master/webmessaging/multi-globals/messageport-current.html

**Review by domenic — APPROVED** [review]
LGTM from a multi-globals perspective, with a potential small simplification.

It would be ideal to get @d0iasm's review as well from a worker/COEP reporting perspective.

File: workers/multi-globals/entry.html (hunk 1 above)
Comment (zcorpan) [reply workers/multi-globals/entry.html]: Fixed

**Comment by d0iasm** [comment]
Sorry for being late. LGTM! Thank you for adding the tests.

**Pull request merged**